components/infobox/commons/infobox_person.lua

```diff
@@ -144,7 +144,10 @@ function Person:createInfobox()
 				}}
 			end}
 		}},
-		Cell{name = 'Alternate IDs', content = {args.ids or args.alternateids}},
+		Cell{name = 'Alternate IDs', content = {
+				table.concat(mw.text.split(args.ids or args.alternateids or '', ',%s?'), ' &#8226; ')
```

Comment:
Option A: Improve by using non-breaking space before dot:
```suggestion
				table.concat(mw.text.split(args.ids or args.alternateids or '', ',%s?'), '&nbsp;&#8226; ')
```
Option B: Concat with comma to keep current display
```suggestion
				table.concat(mw.text.split(args.ids or args.alternateids or '', ',%s?'), ', ')
```

Reply:
i personally like `, ` better

Reply:
Changed to `, `